components/Tile.tsx

```diff
@@ -46,8 +47,22 @@ export default function Tile({ tile }) {
                     alignItems="flex-start"
                 >
                     <span style={{ fontSize: '20px' }}><b>{tile.title}</b></span>
-                    <span style={{ fontSize: '15px' }}><b>${tile.amt}</b></span>
-                    <img id="listingImage" src={tile.thumbnail} alt={tile.title} />
+                    <span style={{
+                        fontSize: '30px',
+                        position: 'absolute',
+                        top: 0,
+                        right: 10,
+                    }}
+                    >
+                        <b>${15}
+                        </b>
+                    </span>
+                    <Image
```

Comment:
Generally you'd need to check that thumbnail exists with { tile.thumbnail && ...} but I will merge for now for a quick fix for launch.